producer/src/main/kotlin/streams/procedures/StreamsPublishResult.kt

```diff
@@ -0,0 +1,12 @@
+package streams.procedures
+
+data class StreamPublishResult(
```

Comment:
```suggestion
data class StreamPublishResult(@JvmField val value)
```

usa sempre `val` dove possibile perché equivale a `final`